poc/Cornelius.tbasic

```diff
@@ -707,27 +710,28 @@ function sleepDisplay()
   logger.wInfo("Now sleeping")
 
 ' -- Main loop
-  While tbgl_IsWindow(hWnd)
+  While IsWindow(hWnd)
     call drawing
-' -- ESCAPE key to exit application
-    If tbgl_GetWindowKeyState(hWnd, %VK_ESCAPE) Then Exit While
+    
+    if asc(Canvas_Inkey)=  27 Then Exit While
   Wend
 
-  tbgl_DestroyWindow
+  Canvas_BitmapSaveImageFile("result.png")
```

Comment:
The output looks correct to me:
![result](https://user-images.githubusercontent.com/8292220/81857648-d6205c00-9562-11ea-8826-1a7ac290eae4.png)

Reply:
This is beyond expectation ! Thanks again !